evaluate.py

```diff
@@ -125,8 +129,14 @@ def pump_it(self):
 
         # TODO: validate the object
         try:
+            # Change the reader config if a source file was specified via args
+            if args.source_xml:
+                reader_file_path_entry = next((x for x in profile["pipeline"]["reader"] if isinstance(x, dict) and x.get("file_path")), None)
```

Comment:
Smells like a ruby :)

Reply:
I was amazed I could do that in Python 😉